project/common.scala

```diff
@@ -88,4 +90,16 @@ object common {
     pomIncludeRepository := { _ => false },
     publishArtifact in Test := false
   )
+
+  def customSettings = Seq(
+    // "0.8.1a" "0.7.3a"
+    scalazStreamVersion := {
+      if(sys.env.get("TRAVIS").nonEmpty){
```

Comment:
because for development you can just do: `set scalazStreamVersion := "0.8.1a"` or whatever